packages/cli/lib/errorHandlers/overrideErrors.js

```diff
@@ -1,38 +1,46 @@
-const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
+const {
+  isSpecifiedError,
+  isMissingScopeError,
+} = require('@hubspot/local-dev-lib/errors/apiErrors');
 const { logger } = require('@hubspot/local-dev-lib/logger');
-
 const { PLATFORM_VERSION_ERROR_TYPES } = require('../constants');
 const { i18n } = require('../lang');
-const { uiLine, uiLink } = require('../ui');
+const {
+  uiAccountDescription,
+  uiLine,
+  uiLink,
+  uiCommandReference,
+} = require('../ui');
 
 const i18nKey = 'lib.errorHandlers.overrideErrors';
 
-function createPlatformVersionError(subCategory, errData) {
-  const docsLink = uiLink(
-    i18n(`${i18nKey}.platformVersionErrors.docsLink`),
-    'https://developers.hubspot.com/docs/platform/platform-versioning'
-  );
-
-  const platformVersionKey = {
-    [PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_NOT_SPECIFIED]:
-      'unspecified platformVersion',
-    [PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED]:
-      errData.context.RETIRED_PLATFORM_VERSION,
-    [PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST]:
-      errData.context.PLATFORM_VERSION,
-  };
+function createPlatformVersionError(err, subCategory) {
+  let translationKey = 'unspecifiedPlatformVersion';
+  let platformVersion = '';
```

Comment:
Any reason to default the `translationKey` but not the `platformVersion`? From what I could tell, you could set this to "unspecified platformVersion" and remove the check for `PLATFORM_VERSION_NOT_SPECIFIED` in the switch

Reply:
Ah, no not really. I updated in [1bfea4b](https://github.com/HubSpot/hubspot-cli/pull/1128/commits/1bfea4bfb993b0b7eba0b9a95387f6a9f784fe97)